### PR TITLE
Adding Category Name to CoreAlertFields

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CRTX-59962

## Description
Adding Category Name field to CoreAlertFields.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Category_Name.json [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
+    "unmapped": false,
+    "unsearchable": true,
+    "caseInsensitive": true,
+    "sla": 0,
+    "threshold": 72,
+    "fromVersion": "6.5.0",
+	"x2_fields": "alert_category",
```
Comment (shaniacht1) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Category_Name.json:29]: I think it's just "category". Why did you add the prefix?

Comment (tomer-pan) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Category_Name.json:29]: It's "alert_category" 100%
A screenshot from the python file with all the fields:
image

I think the issue with this field was that we already had an incident field with the name "Alert Name" in XSOAR (under common types)

**Comment by tomer-pan** [comment]
@michalgold @shaniacht1 After a verification with @dheinisch14 and Sharon Harel Eilam, this is working as expected, please approve this PR so we could merge.
Once merged and uploaded to the MP, customers will see the "Category" value under the "categoryname" incident field in the context.

**Review by michalgold — APPROVED** [review]
